app/grandchallenge/cases/image_builders/metaio_mhd_mha.py

```diff
@@ -97,6 +97,8 @@ def convert_itk_file(
                 timepoints=timepoints,
                 resolution_levels=None,
                 color_space=color_space,
+                pixel_width=None,
+                pixel_height=None,
```

Comment:
Would be useful to have this information for mhd too. This should be something like:

```
                pixel_width=simple_itk_image.GetSpacing()[0],
                pixel_height=simple_itk_image.GetSpacing()[1],
```